openapi/java.xml

```diff
@@ -0,0 +1,66 @@
+<project 
+  xmlns="http://maven.apache.org/POM/4.0.0" 
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>io.kubernetes</groupId>
+  <artifactId>client-java</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <name>client-net</name>
+  <url>http://kubernetes.io</url>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>io.swagger</groupId>
+        <artifactId>swagger-codegen-maven-plugin</artifactId>
+        <version>2.2.2</version>
+        <executions>
+          <execution>
+            <goals>
+              <goal>generate</goal>
+            </goals>
+            <configuration>
+              <inputSpec>${generator.spec.path}</inputSpec>
+              <language>java</language>
+              <gitUserId>kubernetes-incubator</gitUserId>
```

Comment:
s/kubernetes-inclubator/kubernetes-client/

This need to be fixed in other files too. I will fix that.